### PR TITLE
Features/473 row setters

### PR DESCRIPTION
**Pull request by owlz84**
Thanks for contributing.

- [x] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

## Description

- Added setters to Row class for:
  - Ints, Shorts and Longs
  - Floats and Doubles
  - Dates, Times and DateTimes
  - Strings and Text
  - Booleans
- Omitted setters for PackedDate, PackedTime and PackedDateTime
- Omitted setter for Object columns

## Testing

Full set of (very basic) tests for getters and setters in Row. Selected appropriate source data and data types for testing each method.

### CODE REVIEW

File: core/src/test/java/tech/tablesaw/api/RowTest.java [hunk 1]
```diff
@@ -6,8 +6,11 @@
 import java.io.File;
 import java.io.IOException;
 import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static tech.tablesaw.api.ColumnType.*;
```
Comment (lwhite1) [inline core/src/test/java/tech/tablesaw/api/RowTest.java]: Please remove the unused import. Codacy will thank you.

File: core/src/main/java/tech/tablesaw/api/Row.java [hunk 2]
```diff
@@ -82,64 +84,110 @@ public boolean hasNext() {
...
 
-    public float getFloat(String columnName) {
-	return floatColumnMap.get(columnName).getFloat(rowNumber);
+    public Object getObject(int columnIndex) {
+        return columnMap.get(columnNames[columnIndex]).get(rowNumber);
     }
 
-    public float getFloat(int columnIndex) {
-        return getFloat(columnNames[columnIndex]);
+    public int getPackedDate(String columnName) {
```
Comment (benmccann) [inline core/src/main/java/tech/tablesaw/api/Row.java]: I would probably not include getters for the packed date / datetime as I think it's a bit of an implementation detail

Comment (lwhite1) [reply core/src/main/java/tech/tablesaw/api/Row.java]: I think i've used them for performance sensitive work, but i agree they'd almost never be needed in normal use. i'm ok with their being removed.

File: core/src/test/java/tech/tablesaw/api/RowTest.java (hunk 1 above)
Comment (owlz84) [reply core/src/test/java/tech/tablesaw/api/RowTest.java]: Removed in latest commit.

File: core/src/main/java/tech/tablesaw/api/Row.java (hunk 2 above)
Comment (lwhite1) [reply core/src/main/java/tech/tablesaw/api/Row.java]: I'll add an issue for this and do it in a subsequent change

**Pull request merged**